stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -104,34 +104,34 @@ class Element(MutableSequence[Element]):
     text: Optional[_str_result_type]
     tail: Optional[_str_result_type]
     def __init__(self, tag: Union[_str_argument_type, Callable[..., Element]], attrib: Dict[_str_argument_type, _str_argument_type] = ..., **extra: _str_argument_type) -> None: ...
-    def append(self, subelement: Element) -> None: ...
+    def append(self, __subelement: Element) -> None: ...
     def clear(self) -> None: ...
     def copy(self) -> Element: ...
-    def extend(self, elements: Iterable[Element]) -> None: ...
+    def extend(self, __elements: Iterable[Element]) -> None: ...
     def find(self, path: _str_argument_type, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Optional[Element]: ...
     def findall(self, path: _str_argument_type, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> List[Element]: ...
     @overload
-    def findtext(self, path: _str_argument_type, *, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Optional[_str_result_type]: ...
+    def findtext(self, path: _str_argument_type, default: None = ..., namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Optional[_str_result_type]: ...
     @overload
     def findtext(self, path: _str_argument_type, default: _T, namespaces: Optional[Dict[_str_argument_type, _str_argument_type]] = ...) -> Union[_T, _str_result_type]: ...
     @overload
-    def get(self, key: _str_argument_type) -> Optional[_str_result_type]: ...
+    def get(self, key: _str_argument_type, default: None = ...) -> Optional[_str_result_type]: ...
```

Comment:
This change is unnecessary, since it overlaps with the next overload. But since it doesn't harm either, we can leave it in. (Same for two other places in this PR.)